grpc/src/mapper.rs

```diff
@@ -191,6 +191,8 @@ impl TryFrom<AssetDetails> for CompleteAssetDetails {
                 .transpose()?
                 .ok_or(GrpcError::MissingField("owner_type".to_string()))?,
             owner_delegate_seq,
+            is_current_owner: todo!(),
```

Comment:
I suppose it will panic on `try_into()` conversion

Reply:
Yep, grpc will panic: https://linear.app/mplx/issue/MTG-977/propagate-ownership-changes-to-grpc made a ticket for that